### DISCUSSION TITLE
Play button behaviour

### DISCUSSION THREAD

**Discussion by kosukesando** [jeffvli/feishin#876] [Q&A] [answered]
Is there a way to make the play button behave similar to other software (Spotify, Musicbee) where pressing the play button on a song in an album adds the remaining songs in the album to the queue?
It's weird to me that that isn't the default behavior, let alone the fact that there isn't even an option to do that from the right-click menu as well.

**Comment by iiPythonx** [comment, marked as answer]
Check Settings -> General, in the third section:
![image](https://github.com/user-attachments/assets/e7642ba7-17fe-41dd-97aa-a4e55dc82e80)

If you specifically mean you want it to add the *remaining* songs in the album, then no I don't believe there's a way to do that from within Feishin.

**Reply by kosukesando** [reply]
Hm, that's a shame considering all the other things this app does right.

**Comment by Grumley** [comment]
Came here with a similar question, there is now an option to double-click play and all the other tracks will be added in queue. Though using this is unreliable at best.

I don't think pressing play should clear the current queue. https://github.com/jeffvli/feishin/issues/54 this 2023 ticket seems relevant.

<img width="685" height="151" alt="image" src="https://github.com/user-attachments/assets/5beb6e98-4b11-4107-8717-47937a230068" />
